src/panels/developer-tools/service/developer-tools-service.js

```diff
@@ -118,6 +118,11 @@ class HaPanelDevService extends PolymerElement {
           <mwc-button on-click="_callService" raised disabled="[[!validJSON]]"
             >Call Service</mwc-button
           >
+          <template is="dom-if" if="[[_attributes.length]]">
+            <mwc-button on-click="_fillExampleData" raised
```

Comment:
Let's not make it raised, it's not that important. Even would want to move it away from call.

Reply:
I see that you followed @SeanPM5 mockup, but I think that we can do better.

(this whole page really needs an overhaul, but for future us)